Sofascore/Sofascore/Networking/CompletionHandlerClient.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+struct CompletionHandlerClient: ApiClientProtocol {
+    
+    let baseUrl = "https://static-api.sofascore.dev/api"
```

Comment:
Ovo se sad ponavlja u svim ApiClientima, mozda da izdvojis to u BaseApiClient protocol koji ce ostali protokoli nasljedivati, a onda tamo mozes izdvojiti i ovaj dio sa stvaranjem URL-ova jer se to isto ponavlja.

Reply:
Ili to u nekakav helper, vidi sto ti se vise svidja